src/app/home/components/conditions/conditions.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-conditions',
+  templateUrl: './conditions.component.html',
+  styleUrls: ['./conditions.component.scss'],
+})
+export class ConditionsComponent implements OnInit {
+  constructor() {}
```

Comment:
@MaxOvs19 solve issues related to empty functions via eslint [ticket](https://trello.com/c/yHMtAJ9d/7-add-recommended-angular-eslint)